### PR TITLE
Css two column layout

### PR DESCRIPTION
**Pull request by Yi-Jacob**
Screen Shot 2022-01-28 at 4 47 46 PM

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Jacob, all your `column` classes are named after what they are being used on, not their measurement, which is not the way that `column` classes are meant to be structured. Please review `css-layout-classes` and then revise your layout classes in this challenge to match their intended purpose.

**Comment by Yi-Jacob** [comment]
Screen Shot 2022-01-31 at 10 49 15 AM

File: css-two-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,87 @@
+img {
+  height: 125px;
+  width: auto;
+}
+
+button {
+  margin-top: 50px;
+  margin-left: 425px;
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css]: Please redo this without abusing `margin`

Comment (codingcodymiller) [reply css-two-column-layout/styles.css]: I suggest using `text-align` to center the button.

File: css-two-column-layout/styles.css [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+.container {
+  max-width: 1000px;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.column {
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css]: `column-half`?

File: css-two-column-layout/styles.css [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+  padding-right: 3px;
+}
+
+.column1-6 {
+  flex-basis: calc(100% / 6);
+}
+
+.column1-3 {
+  flex-basis: calc(100% / 3);
+  margin-right: 10%;
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css]: `column` classes should only have `flex-basis` or `width` and nothing else. You should make an additional class you can add to the element in addition to the column class.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by Yi-Jacob** [comment]
Screen Shot 2022-01-31 at 4 52 36 PM

**Comment by Yi-Jacob** [comment]
Screen Shot 2022-02-01 at 12 50 36 PM

**Review by codingcodymiller — APPROVED** [review]
Nice work!

**Pull request merged**